src/sparseml/onnx/base.py

```diff
@@ -32,11 +32,7 @@
 
     onnxruntime_err = None
 
-except ModuleNotFoundError as error:
-    raise error.__class__(
-        f"{error.msg}. To fix this error, install sparseml[onnxruntime]."
-    )
-
+except Exception as error:
```

Comment:
Looking good, just tested it and I like how it really pushes the error instantiation as late as possible. However, I would keep the message for the user that points to `sparseml[onnxruntime]` installation. This way the user will know that this is not some random bug off the happy path, but something which we account for and have a solution to.

Reply:
good call, added something in